app/views/application/home.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Vesta</h1>
+<% if current_user.room.present? %>
```

Comment:
So... this is basically the student dashboard, right, or at least the start of one (i.e. #303)? I feel like this route will end up being our unauthenticated home page (e.g. #321) and/or will be switched to using HighVoltage, so I'm not sure why it goes in this issue, tbh.